user/pages/03.commerce2/02.developer-guide/09.promotions/02.create-a-condition/docs.md

```diff
@@ -4,4 +4,222 @@ taxonomy:
     category: docs
 ---
 
+Commerce conditions are used in when creating a Promotion or a Payment Gateway 
+for your store. 
```

Comment:
This reads like it's only for promotions and payment gateways but it's also for shipping. Can we write it so it's more open-ended? Or at least add shipping there?